Templates (Project)/Hamburger/Views/SettingsPage.xaml.cs

```diff
@@ -17,8 +17,11 @@ public SettingsPage()
 
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
-            var index = int.Parse(_SerializationService.Deserialize(e.Parameter?.ToString()).ToString());
-            MyPivot.SelectedIndex = index;
+            if (e.Parameter != null)
+            {
+                var index = int.Parse(_SerializationService.Deserialize(e.Parameter.ToString()).ToString());
```

Comment:
The JsonSerializationService doesn't return null unless you feed it null or it got null from Json.DeserializeObject(). Better to be safe I guess you're right. I will add the extra elvis and update this pull request.